src/components/inputs/reactHookForm/text/DescriptionField.tsx

```diff
@@ -14,17 +14,24 @@ import { useFormContext } from 'react-hook-form'; // Import useFormContext
 import { FieldConstants } from '../../../../utils/constants/fieldConstants';
 import { ExpandingTextField } from './ExpandingTextField';
 
-export function DescriptionField() {
-    const [isDescriptionFieldVisible, setIsDescriptionFieldVisible] = useState(false);
-    const { setValue } = useFormContext();
+interface DescriptionProps {
+    showDescription?: boolean;
+    description?: string;
+}
+export function DescriptionField({ showDescription }: Readonly<DescriptionProps>) {
+    const { setValue, getValues } = useFormContext();
+    const description = getValues(FieldConstants.DESCRIPTION);
+    const [isDescriptionFieldVisible, setIsDescriptionFieldVisible] = useState(
+        showDescription && description && description !== ''
```

Comment:
```suggestion
        !!description
```
try it whether it works?